tests/integration/tests.mk

```diff
@@ -130,18 +131,11 @@ test.integration.kube.presubmit: | $(JUNIT_REPORT)
 	${_INTEGRATION_TEST_WORK_DIR_FLAG} \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_UNIT_TEST_XML))
 
-# Unstable/flaky/new integration tests targeting Kubernetes environment.
-.PHONY: test.integration.kube.unstable
-test.integration.kube.unstable: | $(JUNIT_REPORT)
+# Integration tests that detect race condition for native environment.
+.PHONY: test.integration.race.native
+test.integration.race.native: | $(JUNIT_REPORT)
 	mkdir -p $(dir $(JUNIT_UNIT_TEST_XML))
 	set -o pipefail; \
-	$(GO) test -p 1 ${T} ${TEST_PACKAGES} ${_INTEGRATION_TEST_WORKDIR_FLAG} ${_INTEGRATION_TEST_CIMODE_FLAG} -timeout 30m \
-    --istio.test.select -presubmit,-postsubmit \
- 	--istio.test.env kube \
-	--istio.test.kube.config ${INTEGRATION_TEST_KUBECONFIG} \
-	--istio.test.hub=${HUB} \
-	--istio.test.tag=${TAG} \
-	--istio.test.pullpolicy=${_INTEGRATION_TEST_PULL_POLICY} \
-	${_INTEGRATION_TEST_INGRESS_FLAG} \
-	${_INTEGRATION_TEST_WORK_DIR_FLAG} \
+	$(GO) test -race -p 1 ${T} ${TEST_PACKAGES} -timeout 120m \
```

Comment:
The tests take like 1-5min in my experience (not actually that variable, I just forgot), so we may want to bump the timeout down. Not a big deal though - there are benefits to having a large timeout as well

Reply:
race test change is just a move, not edit.